### PR TITLE
sed normalization delimiters

### PR DESCRIPTION
**Pull request by denji**
(`/`) -> (`|`, `#`, `@`, ..)

### CODE REVIEW

**Comment by praseodym** [comment]
Unfortunately this change doesn't seem to work:
```
Sending build context to Docker daemon    105kB
Step 1/3 : FROM nginx:alpine
 ---> b411e34b4606
Step 2/3 : RUN sed -i -r '    @user\s+nginx;@d;     s@/var/run/nginx.pid@/tmp/nginx.pid@;     s@/var/log/nginx/error.log/stderr@;     s@/var/log/nginx/access.log@/dev/stdout@'     /etc/nginx/nginx.conf &&   sed -i -r 's@listen\s+80;@listen 8080;@; s@/usr/share/nginx/html@/srv@' /etc/nginx/conf.d/default.conf &&   mv /usr/share/nginx/html/* /srv &&   rmdir /usr/share/nginx/html /usr/share/nginx /var/cache/nginx &&   ln -s /tmp /var/cache/nginx
 ---> Running in 26d8077c4b18
sed: unsupported command @
The command '/bin/sh -c sed -i -r '    @user\s+nginx;@d;     s@/var/run/nginx.pid@/tmp/nginx.pid@;     s@/var/log/nginx/error.log/stderr@;     s@/var/log/nginx/access.log@/dev/stdout@'     /etc/nginx/nginx.conf &&   sed -i -r 's@listen\s+80;@listen 8080;@; s@/usr/share/nginx/html@/srv@' /etc/nginx/conf.d/default.conf &&   mv /usr/share/nginx/html/* /srv &&   rmdir /usr/share/nginx/html /usr/share/nginx /var/cache/nginx &&   ln -s /tmp /var/cache/nginx' returned a non-zero code: 1
```

**Comment by denji** [comment]
@praseodym fixed

$ docker build -t test - < Dockerfile
$ docker run --rm test cat /etc/nginx/nginx.conf

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
 FROM nginx:alpine
 RUN sed -i -r '\
     /user\s+nginx;/d; \
-    s/\/var\/run\/nginx.pid/\/tmp\/nginx.pid/; \
-    s/\/var\/log\/nginx\/error.log/stderr/; \
-    s/\/var\/log\/nginx\/access.log/\/dev\/stdout/' \
+    s@/var/run/nginx.pid@/tmp/nginx.pid@; \
+    s@/var/log/nginx/error.log@/stderr@; \
```
Comment (praseodym) [inline Dockerfile]: `/stderr` needs to be `stderr`:
```diff
< error_log  stderr warn;
---
> error_log  /stderr warn;
```

**Review by praseodym — CHANGES_REQUESTED** [review]
Also, could you rebase this branch on top of the current master?

**Comment by denji** [comment]
Rebased `WISVCH/docker-nginx:master`

**Review by praseodym — APPROVED** [review]

**Comment by praseodym** [comment]
Cheers, thanks!

**Pull request merged**